BrightID/src/routes/HomeDrawer.tsx

```diff
@@ -248,6 +242,28 @@ const CustomDrawerContent = (props) => {
           });
         }}
       />
+      <CustomItem
+        focused={state.routeNames[state.index] === 'Groups'}
+        inactiveTintColor={BLACK}
+        inactiveBackgroundColor={WHITE}
+        activeTintColor={WHITE}
+        activeBackgroundColor={ORANGE}
+        label={t('drawer.label.groups')}
+        // style={styles.drawerItem}
+        // labelStyle={styles.labelStyle}
+        icon={({ focused }) => (
+          <GraphQl
+            width={DEVICE_LARGE ? 28 : 24}
+            height={DEVICE_LARGE ? 28 : 24}
+            color={focused ? GREY : BLACK}
+            highlight={focused ? WHITE : ORANGE}
+          />
```

Comment:
We need a dedicated "group" icon here

Reply:
I wanted to ask @damavara to provide that but I forgot.
I do not know how those vector files are created and placed in `src/components/Icons`.
I will mention her in keybase and ask.